AnimateQuotes.js

```diff
@@ -29,46 +28,65 @@ export default function AnimateQuotes(props) {
         delay: QUOTES_IN_DELAY,
         useNativeDriver: true,
       }),
-    ]).start();
-    const endOfAnim = setTimeout(() => {
-      setIsRegistered(false);
-      clearTimeout(endOfAnim);
-    }, Math.max(SVG_OUT_DURATION, QUOTES_IN_DURATION) + QUOTES_IN_DELAY);
+    ]).start(({ finished }) => {
+      setIsQuoteAnimationRunning(false);
+      setIsToggleReady(true);
+      setHasToggledBefore(true);
```

Comment:
Animation callback runs upon animation completion to set isToggleReady back to true for subsequent shake registry and hasToggledBefore to true to determine which animation compositions should be run after this initial run.